src/yunohost/tools.py

```diff
@@ -104,52 +104,45 @@ def tools_adminpw(auth, new_password):
         logger.success(m18n.n('admin_password_changed'))
 
 
-def tools_maindomain(auth, old_domain=None, new_domain=None, dyndns=False):
+def tools_maindomain(auth, new_domain=None):
```

Comment:
Not sure about this ... `new_domain` makes it explicit that this value is meant to be the new main domain ? To me, just `domain` would be ambiguous.